### ISSUE TITLE
training dataset

### ISSUE THREAD

**Issue by tonystarkiss** [newbee-ML/MIFN-Velocity-Picking#2]
I would like to reproduce the work, could you kindly provide the below dataset for me, thanks,  my email is chengyi@nvidia.com

  --- Root
  ​        |___   segy
  ​                                       |___  vel.stk.sgy

  ​                                       |___  vel.pwr.sgy

  ​                                       |___  vel.gth.sgy

  ​              |___   t_v_labels.dat

**Comment by newbee-ML** [comment]
I'm so glad that you are interested in our work!

I'm sorry, but our data is a state secret, so we cannot share it with you.
There is another solution, that you can understand the model totally, and change the pytorch dataloader defined in our codes.

vel.stk = stack gather slices
vel.gth = common-midpoint (CMP) gather
vel.pwr = velocity spectrum
t_v_labels.dat = time-velocity label (manual picking)

more information about the above concepts can be searched in wiki.
